plugins/lxc/lxc_cpu_system

```diff
@@ -64,25 +58,23 @@ if [ "$1" = "config" ]; then
  echo 'graph_vlabel USER_HZ'
  echo 'graph_category lxc'
 
- 
+
  for guest_name in $guest_names;
- do 
+ do
   guest="$(clean_fieldname $guest_name)"
-  echo 'cpu_system_'$guest'.label '$guest_name': System'
+  echo 'cpu_system_'$guest'.label '$guest_name
   echo 'cpu_system_'$guest'.type DERIVE'
   echo 'cpu_system_'$guest'.min 0'
  done
  exit 0
 fi
 
  for guest_name in $guest_names;
- do 
+ do
   guest="$(clean_fieldname $guest_name)"
-  
-  tmp_g=`$f_comm -n $guest_name cpuacct.stat | grep system`
-  tmp_v=`echo $tmp_g | awk '{print($2)}'`
-  echo 'cpu_system_'$guest'.value '$tmp_v
+  echo -n 'cpu_system_'$guest'.value '
+  cat /sys/fs/cgroup/cpuacct/lxc/$guest_name/cpuacct.stat | grep system | awk '{print($2)}'
```

Comment:
please add double-quotes in order to avoid globbing and whitespace isses
See `shellcheck -s dash ...`.